src/movies.js

```diff
@@ -1,13 +1,34 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+function getAllDirectors(movies) {
+    const myArr = movies.filter(function (director) {
```

Comment:
Possibly because I want to get the directors from the movies array, that would be a transformation, so you possibly would use a `.map()` for that one.

One possible piece of code to make it work would be:

```
  const justDirectors = movies.map(function(movie){
    return movie.director
  })
  
  return justDirectors
```

If you paste the previous piece of code inside the `getAllDirectors` function, now I'm creating a new array with only the directors from each movie